### DISCUSSION TITLE
How to turn off segmented downloads in Cyberduck CLI

### DISCUSSION THREAD

**Discussion by natea11** [iterate-ch/cyberduck#14844] [Q&A] [answered]
When I run this from Duck CLI for files over 1GB, the download runs forever.  Mine got up to near 150% and was double the size of the actual file.  I read online where you can turn off Segmented downloads from the GUI, but I need to turn it off from the CLI.  Is there a way to do this?  Thanks

Command I run (I have deleted the actual parameters as they are confidential):  
```
duck --username $accesskey --password $secretkey --download s3://folder/filename $destfolder
```

**Comment by dkocher** [comment, marked as answer]
You can set arbitrary [preferences](https://docs.duck.sh/cli/#preferences) using the command line. To disable segmented downloads run with 

```
env "queue.download.segments=false" duck --username …
```
